package.xml

```diff
@@ -1,20 +1,26 @@
 <?xml version="1.0"?>
-<package>
+<?xml-model
+  href="http://download.ros.org/schema/package_format3.xsd"
+  schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
   <name>node_alive</name>
   <version>0.0.0</version>
-  <description>Node_alive is a watchdog for the nodes that are still alive. 'node_alive_server' checks if all nodes in the node_list are still alive. Nodes can be added to the node_list with use of 'node_alive_add [launch_file]'. 'node_alive_add' scans recursively through all nodes that are being launched with use of the specified launch file and adds these to the node_list.</description>
+  <description>
+    Node_alive is a watchdog for the nodes that are still alive.
+  </description>
 
   <maintainer email="reinzor@gmail.com">Rein Appeldoorn</maintainer>
 
   <license>BSD</license>
 
   <buildtool_depend>catkin</buildtool_depend>
   <build_depend>message_generation</build_depend>
-  <build_depend>std_msgs</build_depend>
 
-  <run_depend>message_runtime</run_depend>
-  <run_depend>std_msgs</run_depend>
-  <run_depend>rospy</run_depend>
-  <run_depend>diagnostic_msgs</run_depend>
+  <depend>diagnostic_msgs</depend>
+  <depend>std_msgs</depend>
+
+  <exec_depend>message_runtime</exec_depend>
+  <exec_depend condition="$ROS_PYTHON_VERSION == 2">python-enum34</exec_depend>
```

Comment:
question: why this dependency on python 2. Would the node-alive software not work with python 3 and up? Or am I misunderstanding this line of code

Reply:
This is a feature from python 3.4. Currently Ubuntu 18.04 is using 3.6, so that feature is included.